conf/broker.conf

```diff
@@ -1380,6 +1380,9 @@ metricsServletTimeoutMs=30000
 # Enable or disable broker bundles metrics. The default value is false.
 exposeBundlesMetricsInPrometheus=false
 
+# Enable or disable compress output metrics in prometheus. The default value is false.
+compressOutputMetricsInPrometheus=false
```

Comment:
Can we make it configurable which type of compression to use instead of making this a boolean?

Reply:
Maybe it's best to add such a configuration when we will add support for more compressions?